install/linux/usr/share/odemis/sim/meteor-tfs2-sim.odm.yaml

```diff
@@ -0,0 +1,207 @@
+# For now, this is esssentially just a FM microscope, as the FIB part is handled
+# separately by the SEM
+
+METEOR-Sim: {
+    class: Microscope,
+    role: meteor,
+}
+
+"SEM": {
+    class: xt_client.SEM,
+    role: null,
+    init: {
+           address: "PYRO:Microscope@localhost:4242", #"PYRO:Microscope@192.168.6.5:4242"
```

Comment:
Does this require running the simulated xt-adapter?

Reply:
no  not needed, I will remove it and incorporate  your yaml file structure in https://github.com/delmic/odemis/pull/2893/files#diff-18ae95404cd6730746a0ed90397cca0f276ec2e213d950a06406e28a5a4d78e4 Then we can have one yaml file that works with both the PRs

Reply:
I am not able to use raw_coordinates to true in stage-bare if I do not use xt-adaptor simulator. I think that should be fine.